Avoid tripping on //caffe2/test/cpp/jit:test_custom_class_registrations with IS_SANDCASTLE

Author: Alexander Grund (TU Dresden)
--- a/torch/testing/_internal/torchbind_impls.py
+++ b/torch/testing/_internal/torchbind_impls.py
@@ -116,8 +116,6 @@ def load_torchbind_test_lib():
 
     if IS_MACOS:
         raise unittest.SkipTest("non-portable load_library call used in test")
-    elif IS_SANDCASTLE or IS_FBCODE:
-        lib_file_path = Path("//caffe2/test/cpp/jit:test_custom_class_registrations")
     elif IS_WINDOWS:
         lib_file_path = find_library_location("torchbind_test.dll")
     else:
